src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -429,12 +438,16 @@
 					ts.url_params.tribe_events_cat = ts.category;
 				}
 
+				// when having plain permalinks
 				if ( td.default_permalinks ) {
-					if( !ts.url_params.hasOwnProperty( 'post_type' ) ){
-						ts.url_params['post_type'] = config.events_post_type;
-					}
-					if( !ts.url_params.hasOwnProperty( 'eventDisplay' ) ){
-						ts.url_params['eventDisplay'] = ts.view;
+					// when not using the shorcode
+					if ( ! $( document.getElementById( 'tribe-events' ) ).is( '.tribe-events-shortcode' ) ) {
```

Comment:
Same as noted above. 

Might be useful to have a function like: `is_event_shortcode()` that you can use here and above to keep things DRY